public/styles/main.css

```diff
@@ -23,35 +63,129 @@ body.flex-page {
   justify-content: flex-start;
   align-items: center;
   padding: 10px;
+  background-color: white;
+}
+
+.side-bar.logo-container {
+  height: 61px;
+  width: 61px;
+  margin-bottom: 1.5rem;
+}
+
+a.side-bar, i.side-bar {
+  height: 50px;
+  width: 50px;
+  padding: 1rem;
+  text-align: center;
+  margin-bottom: 0.5rem;
+
+  &:hover {
+    background-color: #5670f1;
+  }
+}
+
+a.side-bar.connection-icon {
+  border: 1px solid #c9cace;
+  border-radius: 10px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("/public/images/connection.svg");
+}
+
+a.side-bar.notification-icon {
+  border: 1px solid #c9cace;
+  border-radius: 10px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("/public/images/notificatio.svg");
+}
+
+a.side-bar.folder-icon {
+  border: 1px solid #c9cace;
+  border-radius: 10px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("/public/images/folder.svg");
+}
+
+a.side-bar.category-icon {
+  border: 1px solid #c9cace;
+  border-radius: 10px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("/public/images/category.svg");
+}
+
+a.side-bar.settings-icon {
+  border: 1px solid #c9cace;
+  border-radius: 10px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("/public/images/setting.svg");
 }
 
+
 .flex-page.content {
   display: flex;
   flex-direction: column;
-  min-height: 100vh;
 }
 
 .content.header {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 95vw; /* due to column flex-grow: grows height so to void double wrap of els e.g. col/row */
+  background-color: white;
+}
+
+.header.heading {
+  text-align: center;
+  font-size: 2rem;
+  padding: 0.5rem;
+}
+
+.header.nav {
   display: flex;
   flex-direction: row;
-  align-items: center;
-  min-height: 5vh;
+  border-top: #c9cace 0.1px solid;
+
+  width: 100%;
```

Comment:
do we need this width 100%?

Reply:
yes, due to mobile version not having yet a content element, otherwise it leave a little default background color